tests/annealing_box_unittests.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+import numpy as np
+from syllabus.tests import create_cartpole_env
+from syllabus.curricula import AnnealingBoxCurriculum
+
+class TestAnnealingBoxCurriculum(unittest.TestCase):
+
+    def test_annealing_values(self):
+        cartpole_env = create_cartpole_env()
+        curriculum = AnnealingBoxCurriculum(
+            task_space=cartpole_env.task_space,
+            start_values=[0.2, 0.5],
+            end_values=[0.8, 0.9],
+            total_steps=10
+        )
+
+        # Expacted values
+        expected_values = [
+            [0.2, 0.5],
+            [0.328, 0.53],
+            [0.456, 0.56],
+            [0.584, 0.59],
+            [0.712, 0.62],
+            [0.84, 0.65],
+            [0.968, 0.68],
+            [1.096, 0.71],
+            [1.224, 0.74],
+            [1.352, 0.77],
+            [1.48, 0.8]
+        ]
```

Comment:
These expected values seem very wrong? The final values should be 0.8, 0.9

So something like:
[0.20, 0.50]
[0.26, 0.54]
[0.32, 0.58]
[0.38, 0.62]
[0.44, 0.66]
[0.50, 0.70]
[0.56, 0.74]
[0.62, 0.78]
[0.68, 0.82]
[0.74, 0.86]
[0.80, 0.90]
[0.80, 0.90]
[0.80, 0.90]